docs/content/doc/advanced/hacking-on-gitea.en-us.md

```diff
@@ -130,10 +130,10 @@ You should run the same version of go that is on the continuous integration
 server as mentioned above. `make fmt-check` will only check if your `go` would
 format differently - this may be different from the CI server version.
 
-You should lint, vet and spell-check with:
+You should lint, vet, revive and spell-check with:
 
 ```bash
-make vet lint misspell-check
```

Comment:
make lint is depricated. Use "make revive", so we should remove lint

Reply:
To not have to repeat the words in the text I summarized revive + vet to _code analysis_, which I think is a generic name for it.